<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_DlgNewDimStyle</class>
 <widget class="QDialog" name="LC_DlgNewDimStyle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Dimension Style</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="lblUseFor">
     <property name="text">
      <string>Use For:</string>
     </property>
     <property name="buddy">
      <cstring>cbUseFor</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbUseFor">
     <property name="toolTip">
      <string>Defines which type of dimensions are controlled by this style.</string>
     </property>
     <item>
      <property name="text">
       <string>All Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Linear Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Angular Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Radius Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Diameter Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ordinate Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Leader And Tolerances</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblBasedOn">
     <property name="text">
      <string>Based on Style:</string>
     </property>
     <property name="buddy">
      <cstring>cbBasedOnStyle</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbBasedOnStyle">
     <property name="toolTip">
      <string>Existing dimension style which new style is based on</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>New Style Name:</string>
     </property>
     <property name="buddy">
      <cstring>leStyleName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leStyleName">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Name of new dimension style</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbBasedOnStyle</tabstop>
  <tabstop>cbUseFor</tabstop>
  <tabstop>leStyleName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
